include/zephyr/drivers/video-controls.h

```diff
@@ -31,42 +38,91 @@ extern "C" {
 
 /**
  * @name Control classes
+ *
+ * List of control classes, to group related controls together in blocks.
+ *
+ * This list is kept identical to the Linux kernel definitions.
+ *
  * @{
  */
-#define VIDEO_CTRL_CLASS_GENERIC	0x00000000	/**< Generic class controls */
-#define VIDEO_CTRL_CLASS_CAMERA		0x00010000	/**< Camera class controls */
-#define VIDEO_CTRL_CLASS_MPEG		0x00020000	/**< MPEG-compression controls */
-#define VIDEO_CTRL_CLASS_JPEG		0x00030000	/**< JPEG-compression controls */
-#define VIDEO_CTRL_CLASS_VENDOR		0xFFFF0000	/**< Vendor-specific class controls */
+#define VIDEO_CTRL_CLASS_BASE			0x00980900 /**< Base class controls */
+#define VIDEO_CTRL_CLASS_CODEC			0x00990900 /**< Stateful codec controls */
```

Comment:
Same here : 

```
#define V4L2_CTRL_CLASS_CODEC		0x00990000
#define V4L2_CID_CODEC_BASE		(V4L2_CTRL_CLASS_CODEC | 0x900)
```

Reply:
This is now updated `VIDEO_CTRL_CLASS_CODEC_BASE`